Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +17,58 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+function passwordValidation(pass) {
+  let bannedPasswords = ["ydt635RERE!!", "hdud738#&sg", "bxv735#$ls0HG"];
+  if (
+    /[A-Z]/.test(pass) &&
+    /[a-z]/.test(pass) &&
+    pass.length >= 5 &&
+    /[0-9]/.test(pass) &&
+    /[!#$%.*&]/.test(pass) &&
+    bannedPasswords.every((password) => password !== pass) &&
```

Comment:
Can also use the `.include()` method of array.

Reply:
Thanks for your comment!